.github/workflows/update-dependent-repositories-gomod.yaml

```diff
@@ -24,6 +24,7 @@ jobs:
           - cmd-registry-proxy-dns
           - cmd-nse-vfio
           - cmd-nsc-init
+          - cmd-admission-webhook-k8s
```

Comment:
We should also add it for release action.
https://github.com/networkservicemesh/sdk/blob/main/.github/workflows/release.yaml#L46

Reply:
@denis-tingaikin https://github.com/networkservicemesh/sdk/pull/1134